### ISSUE TITLE
A clear example?

### ISSUE THREAD

**Issue by dsernst** [FaceDemocracy/manifesto#8]
Here's a question just so I, as a reader, can understand the intentions and use-case better. Can this document provide a clear example? 

Ideally it would answer each of these:

- [ ] The name of an organization (with a Facebook page) that may be interested in adopting liquid democracy
- [ ] Why the organization decides to install FaceDemocracy
- [ ] How a new ballot item is decided upon and added to the voting agenda. Is it controlled by the Page admins? Does it come from the community?
- [ ] How the community members learn that a new ballot item is being put to a vote
- [ ] Why a community member decides they want to vote
- [ ] Where liquid delegation comes into play
- [ ] The outcome for the organization and its community

Hope these make sense. Cheers 😄

**Comment by diegomrodz** [comment]
@dsernst  Yeah, they made a lot of sense. So, I will answer all the points raised by you and once we get to an understanding I will make a commit to the manifesto adapting it.

I will answer first on a general sense, then I will try to provide some examples that seem valuable to me.

- [ ] **The name of an organization (with a Facebook page) that may be interested in adopting liquid democracy**

I think the organizations that would be most interested on adopting the FaceDemocracy app would be the ones that have a community around it. And the first example that I can think of are universities, where the students are very interested in participating on a more proactive way on the day-to-day of the institution. And I think that hosting a democratic process, right on the facebook page would be very useful because its a familiar environment and would be a showcase its the values.

Like my university, [Centro Universitário Senac](https://www.facebook.com/universitariosantoamaro/), we have a lot of social projects that are organized by the students itself. Where the engineering classes would propose solutions and develop it in order to solve problems for the poor communities around the campus. I have participate in one recently, where we develop this water quality monitoring tool connected to a web application. Because we didn't had a more structured way of discussing and taking decisions, I feel that we ended losing precious time arguing over the same issues over and over again. And also, I think the value of having such tool on Facebook is that people from the communities would be able to participate on the decision making process. So, we would granted passcode for the profile of members of the community to that ballot, and then they would be able to vote, delegate and discuss about it.

_The page is in portuguese, but I would like to present a real-world problem that would be solved by this platform._ 

- [ ] Why the organization decides to install FaceDemocracy

Well, depends a lot on its objetives, it could be a casual ballot or running the organization entire based upon the proposals and issues gathered on the app. I think that we really need to provide a lot of options.

Another example would be a condo, where people have to decide about a lot of issues and at the same time they don't have the time to think about it. Instead of annoying regular meetings, where the ones in charge make everything look boring enough so nobody question it. People would make the decision making part of the dallying routine, delegating votes on specifics issues and voting on others. I think messengers bots are very useful on that, because they may notify people as the ballots are created and the decisions are made.

- [ ] How a new ballot item is decided upon and added to the voting agenda. Is it controlled by the Page admins? Does it come from the community?

Well, I think that ultimate the ballot must be decided by the page's admin, but allowing the community to propose ballot would really be great. What do you think about it?

- [ ] How the community members learn that a new ballot item is being put to a vote

All ballot started as a post on the facebook page, where everyone subscribed would see on their feed. Also, we may send messages via messenger or even emails depending on the importance of the ballot.  

- [ ] Why a community member decides they want to vote

Well, it will vary accordingly on how the organization affects him. I think that in most cases people would vote because of its familiarity with the platform. So, when you see that your university is opening a ballout about something you are interested, then you will probably check it out. I think its important that a vote should be a two step process, where you click on the post and then cast your vote. So people aren't frustrated about the process.

See, people with a lot of interest on the matter will probably want people to delegate their vote, so they could make campaign posts and put a generated link that delegates the vote to them. So their friends could see and delegate the decision on that matter. I think this is true value of the platform being integrated with facebook, Because people naturally express their opinions, and having a democratic way of doing that will be really attractive. 

Also, I think that a gamification of the process isn't a bad idea. Which will make people want to accumulate voting power, which can make liquid democracy addictive, rs.

- [ ] Where liquid delegation comes into play

So,  depends upon the configuration of the ballot, which can be open to everyone, specific to the organization, or specific to that ballot. I think that system of tags like the Google experiment could be valuable as well.  Then, users able to vote on a ballot may be able to delegate, via link generated on the platform, to other users with the same status. Once delegated, you can see all of its voting record. If you don't agreed with something, you can always get change your vote. I think that one day before the voting is ended, I message could be sent by messenger asking you to confirm your delegated vote. The vote still valid, but it will make you have a last chance to look at the discussions and see if your mind has changed.

- [ ] The outcome for the organization and its community

More participation, a sense of openness and respect will surely derive from its application. For a non-profit organization I think that it would help them get new ideas and make better decisions. For a brand, for example, I think that is really great way to see how your customers think about your product.

In the end, the great achievement expected from this project is to introduce the general public to the idea and see how it works.

I am really looking forward to starting developing it, but not before all this issue have been at least discuss about.
